src/views/Room.vue

```diff
@@ -265,4 +268,17 @@ export default {
   width: 100%;
   height: 100%;
 }
+.bubble{
+  height: 25px;
+  width: 25px;
+  left: 10px;
```

Comment:
I do like the styling you've done for the bubble here. Adding a little bit of margin on the left side of the bubble would make it even better.

Since `left` doesn't do anything here, just replace it with margin-left.